extension/src/experiments/columns/collect/index.ts

```diff
@@ -49,22 +50,16 @@ export const collectColumns = (data: ExperimentsOutput): Column[] => {
   return Object.values(acc)
 }
 
-const getData = (value: { baseline: ExperimentFieldsOrError }) =>
-  value.baseline.data || {}
+const getData = (value?: {
+  baseline?: ExperimentFieldsOrError
+}): ExperimentFields | undefined => value?.baseline?.data
 
 export const collectChanges = (data: ExperimentsOutput): string[] => {
   const changes: string[] = []
 
-  let workspace
-  let currentCommit
-
-  for (const [key, value] of Object.entries(data)) {
```

Comment:
Noticed this issue whilst staring at the code for https://github.com/iterative/vscode-dvc/issues/2569#issuecomment-1279053085.

This would not return the correct data now there are > 2 entries in the `exp show` data. It would actually return the oldest commit